question-servers/freeform.js

```diff
@@ -501,6 +611,8 @@ module.exports = {
                         courseIssues.push(courseIssue);
                         return callback(courseIssue);
                     }
+                    delete data.extensions;
+                    delete ret_val.extensions;
```

Comment:
This stops the extensions being passed along to `server.py`?

Reply:
Yeah, I'm thinking only the element code should be able to access it?

Reply:
Yep, makes sense.